source/float/floatItem.css

```diff
@@ -0,0 +1,14 @@
+@keyframes floatItem {
+  from {
+      transform: translate3d(0, 0, 0);
+  }
+  to {
+      transform: translate3d(0, -40px, 0);
+  }
```

Comment:
This could use [some easing](https://cubic-bezier.com/#.17,.67,.83,.67). It's a bit too stiff for a 'float' animation.